test/langtools/jdk/javadoc/doclet/testSingletonLists/TestSingletonLists.java

```diff
@@ -272,9 +263,11 @@ public void docType(String doctype) {
         @Override
         public void startElement(String name, Map<String,String> attrs, boolean selfClosing) {
             switch (name) {
-
                 case "ul": case "ol": case "dl":
                     counts.push(new TreeMap<>());
+                    if ("see-list".equals(attrs.get("class"))) {
+                        inSeeList = true;
+                    }
```

Comment:
This is OK for now, because the lists from `@see` tags cannot be nested. But, I had a similar problem in my work yesterday, and so going forward we might want a more general solution to track the class of each list on the stack.